### ISSUE TITLE
Display annotations by server-specified type

### ISSUE THREAD

**Issue by benwbrum** [ProjectMirador/mirador#3100]
Annotations provided by other systems may fall into categories that do not really fit into `motivation` or other standard parts of the spec.  For example, a manuscript might have annotations representing a diplomatic transcription, an interpretive transcription, a translation, and perhaps an index or commentary.  

If Mirador could display annotations of a single one of these categories alongside an image, we could use it to display a bilingual edition as a triptych, with a panel for the image, a panel for the transcription, and a panel for the translation, or as a toggle between transcription and translation within a single panel.  (See [The Dream of the Rood in EVT](http://evt.labcd.unipi.it/demo/evt_v1-3/dotr/#doc=VB_text_DOTR&page=VB_fol_104v) for an example of a toggle UI in a non-Mirador viewer.)

A simple example of a (v2.1) manifest with a transcript and a translation produced by FromThePage is https://fromthepage.com/iiif/34796/manifest, in which each canvas contains separate annotation lists for transcriptions and translations.  (See [the corresponding document in FromThePage](https://fromthepage.com/harvardlibrary/colonial-north-america-houghton-library/sparks-jared-1789-1866-collector-jared-sparks-collection-of-documents-concerning-the-american-revolution-1740-1866-ternay-charles-louis-d-arsac-de-1722-1780-a-l-s-to-george-washington-n-p-31-jul-1780-ms-sparks).)

A more scholarly example is the [Deiphira Project](https://deiphira.wordpress.com/), which is transcribing and translating [Deiphira on FromThePage](https://fromthepage.com/laurakmorrealellc/humanist-manuscripts/alberti-leon-battista-1404-1472-deiphira-manuscript-not-after-1472-ms-typ-422-houghton-library-harvard-university-cambridge-mass) with a corresponding manifest at  https://fromthepage.com/iiif/33728/manifest .  In this case, using a Mirador workspace as a publication platform (perhaps using a content state that opened panels for transcription and translation?) would be very appealing.

### Modeling options
If Mirador allows this kind of filtering by category, how should annotation providers model their data?

Annotations of _Diplomatic Transcription_, _Interpretive Text_, and _Translation_ categories might have a `motivation` of `commenting`, `supplementing`, or both, but there is no way to use `motivation` to differentiate them from each other.  (Even if there were, it probably shouldn't be the job of the WA spec to enumerate every conceivable scholarly output.)  

One possible option would be to use `purpose` to differentiate annotations (or annotation bodies) from each other in this way.  The down side of `purpose` is that it seems hard to attach human-readable metadata to each category of annotations, which is particularly challenging for internationalized labels.

Perhaps a better way would be to use OA layers (now AnnotationCollections) to aggregate annotations of the same category.  A canvas could have an AnnotationCollection for "Diplomatic Transcription", "Normalized Text", or "Translation", each containing only the annotations of that category.  This would put much of the burden of presentation on the annotation provider (who has the domain knowledge aggregate and label appropriately), so that Mirador would only need to present a AnnotationCollections as options and then display annotations from a collection.

See related issue #2946

**Comment by ColeDCrawford** [comment]
Related to https://github.com/ProjectMirador/mirador/issues/2946

**Comment by marlo-longley** [comment]
- The IIIF model for this has since been settled: https://iiif.io/api/cookbook/recipe/0309-annotation-collection/
- MiradorCanvas.canvasAnnotationPages returns all annotation pages; multiple pages per canvas are loaded
-  all pages render together in the sidebar with no way to distinguish or switch between them

This essentially asking to implement the deferred "Source" dimension from the #2946 annotation filter mockup. The two issues could share a filter panel, but they should stay separate as the architectural work is distinct.

**Comment by marlo-longley** [comment]
This is blocked by #589 whole feature (switch between "Diplomatic Transcription" vs. "Translation" as separate collections) requires knowing that grouping in the first place, which is the data #589 would surface. This feature can't be built without #589's work existing first
